src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,107 @@
+package pages;
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+import utils.Captcha;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class LoginPage {
+    private static String baseUrl = "http://jira.hillel.it:8080/login.jsp";
+    private static SelenideElement element = null;
+    private static String xpathAuiMessageError = "//div[@class='aui-message error']";
+
+    public static void login(String login, String password) {
+        open(baseUrl);
+        setUpFieldsForLogin(login, password);
+        loginButton().click();
+
+        boolean whatchDog = true;
+        while (whatchDog){
```

Comment:
Please, make sure that you don't run into infinite loop. 
You can add number of retries and when they will exceed, then you can stop the loop.